src/app/components/SVG/SvgPopupTriangle.tsx

```diff
@@ -18,7 +18,10 @@ const styles = {
     left-20
     bottom-[-15px]
     w-6
-    drop-shadow-[1px_-1px_1px_#333639]
+    drop-shadow-[1px_-1px_1px_#cfd9de] dark:drop-shadow-[1px_-1px_1px_#333639]
     rotate-180
   `,
+  triangle__g: `
+    text-primaryBg-light dark:text-primaryBg-dark
```

Comment:
Why text is color of background? `primaryBg` (attention to **Bg**) implies that it used for background